### ISSUE TITLE
InstallShield crash

### ISSUE THREAD

**Issue by jschwartzenberg** [stsp/dosemu2#380]
I'm getting this crash with the InstallShield installers from both Civilization 2 expansion CDs. The crash does not occur in PCEM. It happens in either 286/386 win31 mode.
Screenshot:
![dosemu-win31-installshield-crash](https://cloud.githubusercontent.com/assets/877112/26275991/de70dac4-3d6d-11e7-96e2-051415554dc1.png)

-D9+ME boot.log:
[boot.log-D9+ME.gz](https://github.com/stsp/dosemu2/files/1016176/boot.log-D9.ME.gz)

**Comment by stsp** [comment]
How did you install it before?

**Comment by jschwartzenberg** [comment]
I haven't been able to install them before. I tried the expansion installers for the first time now. The game itself can be installed without problems, just both expansions give this crash.

**Comment by stsp** [comment]
Can you test this with japheth's hdpmi?
I suppose you'll need to run `system/krnl386.exe`
instead of `win.com`.

**Comment by jschwartzenberg** [comment]
Is it possible to do this within dosemu? When I run system/krnl386.exe I get a message it can't load mouse.drv.

**Comment by stsp** [comment]
Absolutely.
You need to go to `c:\windows` and
from there type
`system\krnl386.exe`

**Comment by jschwartzenberg** [comment]
Yep, that starts Windows. Doing that with `hdpmi16 -r` before that doesn't seem to make a difference.

**Comment by stsp** [comment]
Ah, you mean hdpmi under dosemu?
No, not possible, try under pcem.

**Comment by stsp** [comment]
So under pcem you'll need to run hdpmi
then `system\krnl386.exe` but not win.com.

**Comment by jschwartzenberg** [comment]
Just tried that. The installer doesn't crash with hdpmi.

**Comment by stsp** [comment]
Then I guess we need japheth to figure
out the problem. :)
Could you try in msdos_ldt.c set XTRA_LDT_LIM to 0?

**Comment by jschwartzenberg** [comment]
I just made this change:
```
-#define XTRA_LDT_LIM (DPMI_page_size * 4)
+#define XTRA_LDT_LIM 0
```
The crash stays the same.

Are you still in touch with Japheth? It seems a lot of people miss him when I search around online.

**Comment by stsp** [comment]
No and his site seems down...
Hope nothing wrong happened.
Try mailing him.

**Comment by mcericdos** [comment]
I vaguely remember that we all wondered what happened and eventually Tom or Georg contacted him offline and found that Japheth is okay, just stopped his DOS activities :-)

**Comment by stsp** [comment]
How did they contact him off-line?
Have they got his physical address somehow?

**Comment by stsp** [comment]
I'll give $200 to anyone (including @Baron-von-Riedesel)
for fixing this. Since it works properly with hdpmi, I suppose
this won't be a big problem for Baron to find a fix. :)

**Comment by stsp** [comment]
I guess @stefand might be interested in
this one too. :) As this is a windows app.

**Comment by jschwartzenberg** [comment]
Maybe it makes sense to introduce a tag for those. There are more open issues related to Windows.

**Comment by stsp** [comment]
Which ones?

**Comment by stefand** [comment]
I don't have this particular game. According to the Wine appdb it installs and runs in Wine though, so this might give some clues

**Comment by jschwartzenberg** [comment]
> Which ones?

You missed #265, #340 and #373.

**Comment by stsp** [comment]
Does this still crash?
